src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -503,7 +512,6 @@ namespace HVACVariableRefrigerantFlow {
         Real64 HRInitialEIRFrac;          // Fractional cooling degradation at the start of heat recovery from cooling mode
         Real64 HREIRTC;                   // Time constant used to recover from initial degradation in cooling heat recovery
         static Real64 CurrentEndTime;     // end time of current time step
-        static Real64 CurrentEndTimeLast; // end time of last time step
```

Comment:
This static local was causing unit test issue when run sequentially using energyplus_tests. Moved to namespace so that clear_state can reset.

Reply:
Hmm, these static variables here in `CalcVRFCondenser` and again in `InitVRF` and `CalcVRFCondenser_FluidTCtrl` are troublesome. They should either be part of the component structs or should be module level variables. It seems at first glance if there's more than one VRF condenser these variables will be wrong, because `CurrentEndTimeLast` will get updated for the first component and then it will be wrong for the next component? And then some of these variables are beings stored in `...HRTimer` and `...HRTime`

So, an alternative hotfix may be to undo this change (leave the main code as-is for now) and use a higher DayOfSimulation on each successive unit test that relies on these times (just two of them at the moment if I'm following correctly) which should trigger a reset. And then post a new issue to refactor this timer stuff without using any static variables.

Reply:
Sorry, I don't have time for this right now. Maybe in a few days or next week.

Reply:
And yes I agree with you.

Reply:
Can't this be merged to get the unit tests running and then rework this when I or others have time? I'm just in a serious time crunch now to get the VRF in airloop/OAsys finished.

Reply:
No, I'll change it shortly.